backend/ingestion/extract_census_data.py

```diff
@@ -1 +1,147 @@
- 
\ No newline at end of file
+import os
+import sys
+import csv
+import logging
+import requests
+from typing import List, Dict
+from dotenv import load_dotenv
+
+########################################################################################
+# SETUP
+########################################################################################
+
+# Enviornment variables
+load_dotenv()
+CENSUS_API_KEY = os.getenv("CENSUS_API_KEY")
+
+# Logging configuration
+logging.basicConfig(
+    level=logging.INFO, format="%(asctime)s - %(levelname)s - %(message)s"
+)
+
+# Mappings
+variable_ids = {"B01001_001E": "total_population"}
+city_fips = {
+    "nyc": {"state_fips": "36", "county_fips": ["061", "047", "081", "005", "085"]},
+    "chicago": {"state_fips": "17", "county_fips": ["031"]},
+    "portland": {"state_fips": "41", "county_fips": ["051"]},
+}
```

Comment:
This looks good, I'm just wandering if this could be obtained through an API call. I think for the scope of the project this is good, I'm just thinking in case of expanding the project where flexibility could be needed.